pkg/cli/interactive_tests/test_missing_log_output.tcl

```diff
@@ -63,6 +63,10 @@ eexpect ":/# "
 
 # Start a regular server
 send "$argv start >/dev/null 2>&1 & sleep 1\r"
+eexpect "\[1\]"
+eexpect ":/# "
+# Give the server some time to start.
+sleep 0.5
```

Comment:
we call `sdnotify.Ready()` in server, so if there's a `NOTIFY_SOCKET` defined in the env, we'll write `READY=1` to it when we're running.

I'm pretty sure there's an off-the-shelf util that can run a cmd and block until that msg is received?

alternately, we can just use our own `start --background` which does that for us.